src/main.js

```diff
@@ -2,7 +2,19 @@ function convertItemsToUnorderedList(listItems) {
   const ulElement = document.createElement("ul");
   listItems.forEach(item => {
     const liElement = document.createElement("li");
-    liElement.textContent = item.name;
+    liElement.classList.add("fs-api-entry");
+    if (item.type === "file") {
```

Comment:
Assuming we are are getting valid input every time, we can ditch the if statement here in favor of a template literal ([learn more](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)):

```javascript
liElement.classList.add(`fs-api-${item.type}`);
```